src/main/java/software/amazon/documentdb/jdbc/calcite/adapter/DocumentDbEnumerable.java

```diff
@@ -1,17 +1,18 @@
 /*
- * Copyright <2021> Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I'm not sure we want to replace the existing license from copying from Calcite code?

Reply:
Missed this one. Will revert this